tests/baselines/reference/reexportedImportDeclarations.js

```diff
@@ -0,0 +1,28 @@
+//// [reexportedImportDeclarations.ts]
+declare class C {
+    method(): any;
+}
+
+declare namespace ns1 {
+    export { C as A1 };
+}
+declare namespace ns2 {
+    import A2 = ns1.A1;
+    namespace ns21 {
+        export { A2 as A3 };
+    }
+}
+declare namespace ns3 {
```

Comment:
Can you make these non ambient..  Instantiation and non instantiated namespace affect if its emitted right? What if this was Interface instead of class ? It should be non instantiated ?